package.json

```diff
@@ -30,7 +30,8 @@
   "dependencies": {
     "babel-runtime": "5.8.24",
     "fbjs": "^0.7.0",
-    "react-static-container": "^1.0.0-alpha.1"
+    "react-static-container": "^1.0.0-alpha.1",
+    "babel-relay-plugin": "0.7.1"
```

Comment:
I think the idea is that people would need to add `babel-relay-plugin` to their project with npm 3, so it should remain (just) a peer dependency. I'd love someone with more npm knowledge here.. 

cc @steveluscher